build-tools/packages/version-tools/.mocharc.json

```diff
@@ -1,6 +1,5 @@
 {
-  "require": ["test/helpers/init.js", "ts-node/register"],
-  "watch-extensions": ["ts"],
+  "require": ["src/test/helpers/init.js"],
```

Comment:
Fluid-Build noticed that 'src/tests/tsconfig.json' was not built by "npm run build".

I changed the project to compile tests and use the built output rather than invoke 'ts-node'.

Reply:
There was a reason I couldn't put the tests in /src... but I can't remember what it was. :( It's possible that this change will break making local changes then running `/bin/dev` without recompiling?

Reply:
'ts-node' seems happy.  Perhaps in the past the root 'tsconfig.json' file did not exclude '/src/test'?